LayoutTests/platform/wpe/TestExpectations

```diff
@@ -605,6 +605,7 @@ webkit.org/b/204001 pointer-lock/ [ Skip ]
 webkit.org/b/204001 fast/shadow-dom/pointerlockelement-in-shadow-tree.html [ Skip ]
 webkit.org/b/204001 fast/shadow-dom/pointerlockelement-in-slot.html [ Skip ]
 webkit.org/b/204001 imported/w3c/web-platform-tests/pointerevents/pointerlock/pointerevent_coordinates_when_locked.html [ Skip ]
+webkit.org/b/204001 imported/w3c/web-platform-tests/pointerlock/ [ Skip ]
```

Comment:
The expectation on 607 is redundant now that we're skipping the directory.

Reply:
I don't think so. Line 607 is:

`imported/w3c/web-platform-tests/pointerevents/pointerlock`

Line 608 is skipping

`webkit.org/b/204001 imported/w3c/web-platform-tests/pointerlock/`

They don't have the same prefix.

However, there were a few further down the file in `imported/w3c/web-platform-tests/pointerlock/` that were being marked as Failure that I will remove in my next push, since that's now redundant (none of these tests work in wpe).

Reply:
You're right, sorry!